craft_parts/parts.py

```diff
@@ -155,7 +155,7 @@ def get_scriptlet(self, step: Step) -> Optional[str]:
             return self.override_build
         if step == Step.STAGE:
             return self.override_stage
-        if step == step.PRIME:
+        if step == Step.PRIME:
```

Comment:
This is an interesting bug, how did linters let this pass?

Reply:
I'm not entirely sure — I guess because functionally it wasn't a bug (`<instance>.PRIME == <class>.PRIME`, though strictly `<instance>.PRIME is <class>.PRIME.value`)?